modules/maven/antenna-mvn-dependency-tree-analyzer/src/main/java/org/eclipse/sw360/antenna/workflow/analyzers/MvnDependencyTreeAnalyzer.java

```diff
@@ -80,7 +80,7 @@ private ArtifactIdentifier getMavenCoordinates(DependencyNode node) {
 
     @Override
     public String getName() {
-        return "MVN DEPENDENCY TREE";
+        return "Maven dependency anaylzer";
```

Comment:
Why the renaming?

Reply:
I thought it would be good to reflect the name put in the Workflow Step just for consistency. Am not married to the idea though, so if you think it would be better to leave it as it was, I would change it again